isis/scripts/test_IsisDownload.py

```diff
@@ -0,0 +1,52 @@
+from functools import total_ordering
+from tkinter.filedialog import test
+import pytest
+import os
+import json
+import argparse
+from downloadIsisData import rclone 
+
+mission_arr =  ["tgo", "dawn", "cassini", "hayabusa2", "juno", "odyssey", "mro", "mex", "apollo15", "apollo16", "apollo17", "base", "hayabusa", "chandrayaan1", "clementine1", "kaguya", "mariner10", "messenger", "mgs", "near", "newhorizons", "osirisrex", "rosetta", "smart1", "viking1", "viking2", "voyager1", "voyager2"]
+@pytest.mark.parametrize("mission_name",mission_arr)
+def test(mission_name):
+    cfg_path = (os.path.dirname(__file__) + '/../config/rclone.conf')
+
+    # configured to my local path, you will need to change this to test on other machines
+    dataPath= f"/Volumes/common/pkgs/isis3/isis_data/{mission_name}"
```

Comment:
local file paths are not allowed. Also this needs to be something that any dev can run and have pass with just pulling the repo.

Reply:
This is what we temporarily used to test things, @gsn9 is currently working on the parameterized pytest conversion of this.